package.json

```diff
@@ -2,6 +2,11 @@
   "name": "undici",
   "version": "3.3.1",
   "description": "An HTTP/1.1 client, written from scratch for Node.js",
+  "files": [
+    "index.(js|d.ts)",
+    "lib",
+    "docs"
+  ],
```

Comment:
We are missing the `types` directory to I think, right @Ethan-Arrowood ?

Reply:
I forgot types weren't bundled in `lib`, my apologies. Resolved in latest commit.